internal/pkg/template/templates/workloads/partials/cf/autoscaling.yml

```diff
@@ -80,4 +80,161 @@ AutoScalingPolicyECSServiceAverageMemoryUtilization:
+
+BacklogPerTaskScheduledRule:
+  Metadata:
+    'aws:copilot:description': "A trigger to invoke the BacklogPerTaskCalculator Lambda function every minute"
+  DependsOn:
+    - BacklogPerTaskCalculatorLogGroup # Ensure log group is created before invoking.
+  Type: AWS::Events::Rule
+  Properties:
+    ScheduleExpression: "rate(1 minute)"
+    State: "ENABLED"
+    Targets:
+      - Arn: !GetAtt BacklogPerTaskCalculatorFunction.Arn
+        Id: "BacklogPerTaskCalculatorFunctionTrigger"
+
+PermissionToInvokeBacklogPerTaskCalculatorLambda:
+  Type: AWS::Lambda::Permission
+  Properties:
+    FunctionName: !Ref BacklogPerTaskCalculatorFunction
+    Action: lambda:InvokeFunction
+    Principal: events.amazonaws.com
+    SourceArn: !GetAtt BacklogPerTaskScheduledRule.Arn
+
+AutoScalingPolicyEventsQueue:
+  Metadata:
+    'aws:copilot:description': "An autoscaling policy to maintain {{.Autoscaling.QueueDelay.AcceptableBacklogPerTask}} messages/task for EventsQueue"
+  Type: AWS::ApplicationAutoScaling::ScalingPolicy
+  Properties:
+    PolicyName: !Join ['-', [!Ref WorkloadName, BacklogPerTask, !GetAtt EventsQueue.QueueName]]
+    PolicyType: TargetTrackingScaling
+    ScalingTargetId: !Ref AutoScalingTarget
+    TargetTrackingScalingPolicyConfiguration:
+      ScaleInCooldown: 120
+      ScaleOutCooldown: 60
+      CustomizedMetricSpecification:
+        Namespace: !Sub '${AppName}-${EnvName}-${WorkloadName}'
+        MetricName: BacklogPerTask
+        Statistic: Average
+        Dimensions:
+          - Name: QueueName
+            Value: !GetAtt EventsQueue.QueueName
+        Unit: Count
+      TargetValue: {{.Autoscaling.QueueDelay.AcceptableBacklogPerTask}}
+
+{{- $acceptableBacklog := .Autoscaling.QueueDelay.AcceptableBacklogPerTask }}
+{{- range $topic := .Subscribe.Topics}}
+{{- if $topic.Queue}}
+AutoScalingPolicy{{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue:
+  Metadata:
+    'aws:copilot:description': "An autoscaling policy to maintain {{$acceptableBacklog}} messages/task for {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue"
+  Type: AWS::ApplicationAutoScaling::ScalingPolicy
+  Properties:
+    PolicyName: !Join ['-', [!Ref WorkloadName, BacklogPerTask, !GetAtt {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue.QueueName]]
+    PolicyType: TargetTrackingScaling
+    ScalingTargetId: !Ref AutoScalingTarget
+    TargetTrackingScalingPolicyConfiguration:
+      ScaleInCooldown: 120
+      ScaleOutCooldown: 60
+      CustomizedMetricSpecification:
+        Namespace: !Sub '${AppName}-${EnvName}-${WorkloadName}'
+        MetricName: BacklogPerTask
+        Statistic: Average
+        Dimensions:
+          - Name: QueueName
+            Value: !GetAtt {{logicalIDSafe $topic.Service}}{{logicalIDSafe $topic.Name}}EventsQueue.QueueName
+        Unit: Count
+      TargetValue: {{$acceptableBacklog}}
+{{- end }}
+{{- end }}
+
+{{- end }}
```

Comment:
```suggestion
{{- end /*range $topic := .Subscribe.Topics/*}}
{{- end /*if $topic.Queue*/}}

{{- end /*if (and .Subscribe .Autoscaling.QueueDelay)/*}}
```
Maybe 💭

Reply:
Done! apparently you can't have `{{- end /* comment */}}` and it has to be `{{- end }}{/* comment */}}`